### DISCUSSION TITLE
Adding a few more disciplines

### DISCUSSION THREAD

**Discussion by chakrabortydeepro** [PanditProject/Website#24] [Help] [answered]
Would it be possible to add a few more disciplines:
(a) Śikṣā (b) Prātiśākhya (c) Veda (d) Vedalakṣaṇa (e) Pāṇinīya (we have Kātantra as a discipline, so why not Pāṇinīya) (f) Dhātupāṭha (g) Gaṇapāṭha (h) Navya Nyāya (i) Samanvaya

**Comment by ybronner** [comment, marked as answer]
Done. Thanks for the good work, Deepro.

**Comment by chakrabortydeepro** [comment]
Thanks a lot, Prof. Bronner.
